charts/application-core/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 4.3.0
+version: 4.3.1
```

Comment:
this version already exists in the OCI registry... maybe go to 4.4.0

Reply:
Good Catch.